storage/src/adb/any/sync/client.rs

```diff
@@ -1404,8 +1404,8 @@ pub(crate) mod tests {
                 "Expected at least one batch to be processed"
             );
 
-            // Modify the target database by adding 15 more operations
-            let additional_ops = create_test_ops(15);
+            // Modify the target database by adding more operations
```

Comment:
Instead of just changing this bound, I recommend keeping both.

Reply:
Turned this into a table test, which includes this regression case

Reply:
LOVE this